tests/test-cases.js

```diff
@@ -13,6 +13,7 @@ var UNIT_SEP = String.fromCharCode(31);
 var FILES_ENABLED = false;
 try {
 	new File([""], ""); // eslint-disable-line no-new
+	new FileReaderSync(); // eslint-disable-line no-new
```

Comment:
Node 20 added a File global, but not FileReaderSync or FileReader. We need those or else tests fail when we try to instantiate a FileReaderSync: https://github.com/gabegorelick/PapaParse/blob/be7e72cfd4baf3c25d8631b5823e1b4ae88364be/papaparse.js#L667-L681